pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -53,6 +56,10 @@ class EarlyStopping(Callback):
             monitored has stopped decreasing and in ``'max'`` mode it will stop when the quantity
             monitored has stopped increasing.
         strict: whether to crash the training if `monitor` is not found in the validation metrics.
+        check_finite: Stops training when the monitor becomes NaN or infinite. Set this argument to ``False``
+            if this behavior is undesired.
```

Comment:
do we want an option to turn this on/off at all?

Reply:
I would say off. We will add support for occasional NaN loss training soon.